packages/react/src/Navbar.js

```diff
@@ -69,28 +82,43 @@ function Navbar({ children, portalConfig, loginConfig, userDropdownConfig }) {
   if (userDropdownConfig && userDropdownConfig.show) {
     userDropdown = (
       <Dropdown className="user-dropdown" title="User Info" align="end">
-        <Dropdown.Toggle as={Nav.Link} className="profile-dropdown-toggle">
-          <span className="user-icon" alt="Account Information"></span>
+        <Dropdown.Toggle
+          as={Nav.Link}
+          className="profile-dropdown-toggle user-initials"
+        >
+          {userDropdownConfig.initials ? (
+            <span>{userDropdownConfig.initials.toUpperCase()}</span>
+          ) : (
+            <i className="fa fa-user" />
+          )}
         </Dropdown.Toggle>
         <Dropdown.Menu className="profile-dropdown-menu" show={show}>
           <ul className="list-unstyled">
             {userDropdownConfig.additionalItems}
             {userDropdownConfig.logoutLink && (
               <li>
-                <Dropdown.Item 
-                as={Link}
-                to={userDropdownConfig.logoutLink}>
+                <Dropdown.Item as={Link} to={userDropdownConfig.logoutLink}>
                   {userDropdownConfig.logoutMessage || "Sign out"}
                 </Dropdown.Item>
               </li>
-            )
-            }
+            )}
           </ul>
         </Dropdown.Menu>
       </Dropdown>
     );
   }
 
+  let navbarLinks;
+  if (navLinks && navLinks.length > 0) {
+    navbarLinks = (
+      <Nav className="nav-links" as="ul">
+        {navLinks.map((navLink) => (
+          <Nav.Link href={navLink.href}>{navLink.content}</Nav.Link>
+        ))}
+      </Nav>
+    );
+  }
+
```

Comment:
If these are intended to link to in-app routes, we need to allow the `Nav.Link` to render something like `<Nav.Link as={Link}  to={theRoute} ... >` instead of just `href`. I believe `href` is going to cause full page reloading which is not desirable for SPAs.

Reply:
ah yes, good point